app/assets/stylesheets/application.css.scss

```diff
@@ -16,9 +16,38 @@
  *= require_self
 
  */
+$neutral: #d9d9d9;
 
 @import "font-awesome";
 
 legend a {
   font-weight: lighter;
 }
+
+.side-column {
+  font-weight: lighter;
+  text-align: center;
+  ul {
+    width: 70%;
+    margin: auto;
+  };
+  li {
+    list-style: none;
+    // border: 1px solid $neutral;
+    box-shadow: 0 1px 0 #ccc,
+               0 2px 0 #c9c9c9,
+               0 3px 0 #bbb,
+               0 4px 0 #b9b9b9,
+               0 5px 0 #aaa,
+               0 6px 1px rgba(0,0,0,.1),
+               0 0 5px rgba(0,0,0,.1),
+               0 1px 3px rgba(0,0,0,.3),
+               0 3px 5px rgba(0,0,0,.2),
+               0 5px 10px rgba(0,0,0,.25),
+               0 10px 10px rgba(0,0,0,.2),
+               0 20px 20px rgba(0,0,0,.15);
```

Comment:
`.15` should be written with a leading zero as `0.15`<br>Commas in function arguments should be followed by a single space